### PR TITLE
163 disable search while searching

### PR DESCRIPTION
**Pull request by fredkingham**
While the patient is searching for a name/MRN we disable the search button.

Screenshot 2023-02-27 at 18 10 38

### CODE REVIEW

**Comment by fredkingham** [comment]
The failure is solved by https://github.com/openhealthcare/opal/pull/2024

**Comment by davidmiller** [comment]
You sure the imports PR is pointing at a branch that will affect this?

File: opal/core/search/templates/search.html [hunk 1]
```diff
@@ -10,14 +10,15 @@ <h2>
...
-      <form ng-submit="search()" class="form form-horizontal">
+      <form ng-submit="!searching && search()" class="form form-horizontal">
         <div class="row content-offset">
             <div class="col-sm-4 col-sm-push-4">
               <input type="text" autofocus
                      name="name"
                      slash-key-focus="!state || state==='normal'"
                      ng-blur="enableShortcuts()"
                      ng-focus="disableShortcuts()"
+                     ng-change="searching=false"
```
Comment (davidmiller) [inline opal/core/search/templates/search.html:22]: I think I want a comment here to highlight what this does - e.g. re-enables the search button if you change the search string.
I don't 100% know how to comment syntax inside indented single html element that won't break rendering/parsing

**Review by davidmiller — APPROVED** [review]
Observing users, only really seems to matter when a server is slow, but useful then...

**Pull request merged**